### ISSUE TITLE
Docs with same names overwrite themselves

### ISSUE THREAD

**Issue by daguar** [codeforamerica/clean#327]
Uploaded verification documents often have the same name (like img.png). When this happens, they overwrite each other so HSA doesn't get copies of all the docs. @daguar already has a fix for this but not in production yet.

Check full details on Airbrake: [#1353839411039640161](https://herokuapp29009168herokucom.airbrake.io/projects/108305/groups/1353839411039640161)

Backtrace:
```
[PROJECT_ROOT]/app/controllers/documents_controller.rb:35:in +
[PROJECT_ROOT]/app/controllers/documents_controller.rb:35:in block in submit
[PROJECT_ROOT]/app/controllers/documents_controller.rb:29:in each
[PROJECT_ROOT]/app/controllers/documents_controller.rb:29:in submit
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_controller/metal/implicit_render.rb:4:in send_action
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/abstract_controller/base.rb:198:in process_action
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_controller/metal/rendering.rb:10:in process_action
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/abstract_controller/callbacks.rb:20:in block in process_action
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/callbacks.rb:117:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/callbacks.rb:117:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/callbacks.rb:169:in block in halting
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/callbacks.rb:92:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/callbacks.rb:92:in _run_callbacks
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/callbacks.rb:734:in _run_process_action_callbacks
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/callbacks.rb:81:in run_callbacks
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/abstract_controller/callbacks.rb:19:in process_action
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_controller/metal/rescue.rb:29:in process_action
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_controller/metal/instrumentation.rb:31:in block in process_action
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/notifications.rb:164:in block in instrument
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/notifications/instrumenter.rb:20:in instrument
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/notifications.rb:164:in instrument
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_controller/metal/instrumentation.rb:30:in process_action
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_controller/metal/params_wrapper.rb:250:in process_action
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activerecord-4.2.0/lib/active_record/railties/controller_runtime.rb:18:in process_action
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/abstract_controller/base.rb:137:in process
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionview-4.2.0/lib/action_view/rendering.rb:30:in process
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_controller/metal.rb:195:in dispatch
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_controller/metal/rack_delegation.rb:13:in dispatch
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_controller/metal.rb:236:in block in action
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/routing/route_set.rb:73:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/routing/route_set.rb:73:in dispatch
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/routing/route_set.rb:42:in serve
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/journey/router.rb:43:in block in serve
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/journey/router.rb:30:in each
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/journey/router.rb:30:in serve
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/routing/route_set.rb:802:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/rack-1.6.0/lib/rack/etag.rb:24:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/rack-1.6.0/lib/rack/conditionalget.rb:38:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/rack-1.6.0/lib/rack/head.rb:13:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/middleware/params_parser.rb:27:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/middleware/flash.rb:260:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/rack-1.6.0/lib/rack/session/abstract/id.rb:225:in context
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/rack-1.6.0/lib/rack/session/abstract/id.rb:220:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/middleware/cookies.rb:560:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activerecord-4.2.0/lib/active_record/query_cache.rb:36:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activerecord-4.2.0/lib/active_record/connection_adapters/abstract/connection_pool.rb:647:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/middleware/callbacks.rb:29:in block in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/callbacks.rb:88:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/callbacks.rb:88:in _run_callbacks
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/callbacks.rb:734:in _run_call_callbacks
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/callbacks.rb:81:in run_callbacks
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/middleware/callbacks.rb:27:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/middleware/remote_ip.rb:78:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/middleware/debug_exceptions.rb:17:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/middleware/show_exceptions.rb:30:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/railties-4.2.0/lib/rails/rack/logger.rb:38:in call_app
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/railties-4.2.0/lib/rails/rack/logger.rb:20:in block in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/tagged_logging.rb:68:in block in tagged
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/tagged_logging.rb:26:in tagged
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/tagged_logging.rb:68:in tagged
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/railties-4.2.0/lib/rails/rack/logger.rb:20:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/middleware/request_id.rb:21:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/rack-1.6.0/lib/rack/methodoverride.rb:22:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/rack-1.6.0/lib/rack/runtime.rb:18:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/activesupport-4.2.0/lib/active_support/cache/strategy/local_cache_middleware.rb:28:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/middleware/static.rb:113:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/rack-1.6.0/lib/rack/sendfile.rb:113:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/actionpack-4.2.0/lib/action_dispatch/middleware/ssl.rb:24:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/railties-4.2.0/lib/rails/engine.rb:518:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/railties-4.2.0/lib/rails/application.rb:164:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/rack-1.6.0/lib/rack/lock.rb:17:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/rack-1.6.0/lib/rack/content_length.rb:15:in call
[PROJECT_ROOT]/vendor/bundle/ruby/2.1.0/gems/rack-1.6.0/lib/rack/handler/webrick.rb:89:in service
[PROJECT_ROOT]/vendor/ruby-2.1.5/lib/ruby/2.1.0/webrick/httpserver.rb:138:in service
[PROJECT_ROOT]/vendor/ruby-2.1.5/lib/ruby/2.1.0/webrick/httpserver.rb:94:in run
[PROJECT_ROOT]/vendor/ruby-2.1.5/lib/ruby/2.1.0/webrick/server.rb:295:in block in start_thread
```

**Comment by daguar** [comment]
This is depressing, but cool.

**Comment by daguar** [comment]
Fixed in #334
